### PR TITLE
DAOS-5563 tests: Split the findcmd test into two separate tests

### PR DESCRIPTION
**Pull request by berserk-fury**
The functional test 'findcmd features are split into two tests
follows:

* findcmd
  Remove the optional parameters 'sample' and 'challenger_path'.
  The test will fail if find(1) crashes or if the number of files
  found doesn't match the number of files created.
* findcmd_perf
  The test findcmd is executed 'sample' times on DAOS and on the
  challenger file system pointed by the 'challer_path' parameter.
  The test will fail if DAOS performance is lower than its
  challenger.

Signed-off-by: Jonathan Martinez Montes <jonathan.martinez.montes@intel.com>

### CODE REVIEW

File: src/tests/ftest/io/dfuse_find_cmd.py [hunk 1]
```diff
@@ -47,22 +48,50 @@ def test_findcmd(self):
...
+    def _test_findcmd(self, samples=1, challenger_path=""):
+        """
+        Create some number of POSIX containers using dfuse and link them
+        together such that they form a tree with a common root directory. Then,
+        run the find(1) command to locate specific files in various parts of
+        the tree. If the parameter challenger_path is provided, the test will
+        run on it and compare the performance of the provided file system
+        against DAOS File System. The test will run sample times and will
+        maximum data points.
+        """
```
Comment (shimizukko) [inline src/tests/ftest/io/dfuse_find_cmd.py:87]: It would be better if you can provide the parameter descriptions.

**Review by shimizukko — DISMISSED** [review]
LGTM

**Review by shimizukko — APPROVED** [review]
LGTM
You might want to fix the typo in the title: "to separated tests"

**Review by cibervicho — APPROVED** [review]

File: src/tests/ftest/io/dfuse_find_cmd.yaml [hunk 2]
```diff
@@ -22,8 +22,10 @@ container:
...
   cont_count: 5
 find_cmd:
-  samples: 3
   height: 3
   subdirs_per_node: 6
   files_per_node: 10
   needles: 100
+perf:
+  samples: 3
+  challenger_path: /mnt/lustre
```
Comment (bfaccini) [inline src/tests/ftest/io/dfuse_find_cmd.yaml:31]: Well, does it run over Lustre by default ?

Comment (berserk-fury) [reply src/tests/ftest/io/dfuse_find_cmd.yaml:31]: The perf test is disabled :p
`@skipForTicket("DAOS-6530")`

It was suggested to compare DAOS against Lustre. AFAIK we don't have Lustre support on CI. So, I implemented the perf test and wrote "/mnt/lustre" path as an example. DAOS-6530 will enable the perf test.

**Pull request merged**